### ISSUE TITLE
Inherited properties

### ISSUE THREAD

**Issue by briananderton-il** [ababik/Remute#16]
One lacking feature that prevents our adoption of this nice library is support for inherited properties.  Consider these failing tests:

```
	public class Foo
	{
		public Foo(string a)
		{
			A = a;
		}

		public string A { get; }
	}
	
	public class Bar : Foo
	{
		public Bar(string a, string b) : base(a)
		{
			B = b;
		}

		public string B { get; }
	}

	public class Tests
	{
		[Fact]
		public static void RemuteInheritance()
		{
			var bar = new Bar("1", "2");

			// throws System.InvalidCastException : Unable to cast object of type 'Tests.Foo' to type 'Tests.Bar'
			bar = bar.Remute(b => b.A, "3"); 

			Assert.Equal("3", bar.A);
		}

		[Fact]
		public static void RemuteInheritanceAttemptedWorkaround()
		{
			var bar = new Bar("1", "2");
			var config = new ActivationConfiguration()
				.Configure<Bar>(b => new Bar(b.A, b.B));
			var remute = new Remute(config);

			// throws System.Exception : Invalid property 'A'. Must be a member of 'Tests.Bar'
			bar = bar.Remute(b => b.A, "3", remute); 
			
			Assert.Equal("3", bar.A);
		}
	}
```

**Comment by ababik** [comment]
Thanks for reporting this. This shouldn't be a big deal to fix. I'll do it later today.

**Comment by ababik** [comment]
I've released v1.2.3 that addresses this issue.
Please let me know if you have any issues.
